notebooks/official/experiments/comparing_local_trained_models.ipynb

```diff
@@ -0,0 +1,1070 @@
+{
```

Comment:
<p>Suggest under what you learn to include:</p><p> </p><p>Comparing two experiments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/636/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>